app.json

```diff
@@ -2,11 +2,20 @@
   "name": "kingsmen-ah-backend",
   "description": "A community of like minded authors that fosters inspiration and innovation by leveraging the modern web.",
   "scripts": {},
```

Comment:
include the following script 
    `"postdeploy": "heroku config:set HOST_NAME=${HEROKU_APP_NAME}.herokuapp.com --app ${HEROKU_APP_NAME} && npm run refreshdb"`
 we would look more into it later

Reply:
Ok Thanks